src/hardware/serialport/softmodem.h

```diff
@@ -61,105 +61,102 @@ bool MODEM_ReadPhonebook(const std::string &filename);
 
 class CFifo {
```

Comment:
There's uncanny parallels between this and the `Fifo` class defined in `include/serialport.h`.
Variable names, functions, and structure are essentially the same (perhaps the softmodem author was afraid to add a bit more logic to 'Fifo' to support `softmodem`, and just mass-copied it instead?). 
In anycase, it's ripe to merge them - but I feel it's outside the scope of this simple cleanup series.